src/System.Private.ServiceModel/tests/Scenarios/Binding/Http/NetHttpsBindingTests.cs

```diff
@@ -0,0 +1,116 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+using System;
+using System.ServiceModel;
+using System.ServiceModel.Channels;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+
+public static class Binding_Http_NetHttpsBindingTests
+{
+    [Fact]
+    [OuterLoop]
+    public static void StandardUsage_NetHttps_Echo_RoundTrips_String()
+    {
+        string testString = "Hello";
+        ChannelFactory<IWcfService> factory = null;
+        IWcfService serviceProxy = null;
+
+        try
+        {
+            // *** SETUP *** \\
+            NetHttpsBinding netHttpsBinding = new NetHttpsBinding();
+            factory = new ChannelFactory<IWcfService>(netHttpsBinding, new EndpointAddress(Endpoints.HttpBaseAddress_NetHttps));
+            serviceProxy = factory.CreateChannel();
+
+            // *** EXECUTE *** \\
+            string result = serviceProxy.Echo(testString);
+
+            // *** VALIDATE *** \\
+            Assert.NotNull(result);
+            Assert.Equal(testString, result);
+
+            // *** CLEANUP *** \\
+            factory.Close();
+            ((ICommunicationObject)serviceProxy).Close();
+        }
+        finally
+        {
+            // *** ENSURE CLEANUP *** \\
+            ScenarioTestHelpers.CloseCommunicationObjects((ICommunicationObject)serviceProxy, factory);
+        }
+    }
+
+    [Fact]
+    [OuterLoop]
+    public static void TransportWithMessageCredential_NotSupported_NetHttps()
```

Comment:
Yeah, I think until we have an attribute or something like that to mark these kinds of tests, just having the convention of having _NotSupported_ in the name seems like the best approach -- like you have here.